index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+  <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL"
+    crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="theme.css">
+</head>
+
+<body data-bs-theme="dark" class="d-flex flex-column min-vh-100">
+
+  <nav class="navbar bg-body-tertiary">
+    <div class="container">
+      <span class="navbar-brand mb-0 h1">CSS Frameworks Introduction</span>
+    </div>
+  </nav>
+
+  <main class="container flex-grow-1">
+
+    <button class="btn btn-primary">Primary</button>
+    <button class="btn btn-secondary">Secondary</button>
+
+    <ul class="list-group mt-5">
+      <li class="list-group-item">
+        <input type="checkbox" class="form-check-input me-1">
+        <label for="" class="form-check-label">Basic setup of Bootstrap with CDN links</label>
+      </li>
+      <li class="list-group-item">
+        <input type="checkbox" class="form-check-input me-1">
+        <label for="" class="form-check-label">Key Bootstrap Components and Utilities</label>
+      </li>
+      <li class="list-group-item">
+        <input type="checkbox" class="form-check-input me-1">
+        <label for="" class="form-check-label">Basic customisation of Bootstrap with CSS variables</label>
```

Comment:
Labels should match a valid input ID.

```suggestion
        <label for="exampleInputID" class="form-check-label">Basic customisation of Bootstrap with CSS variables</label>
```